ops/cfn/vpc.yml

```diff
@@ -0,0 +1,429 @@
+# Creates custom VPC across 2 AZs, 2 public and 4 private subnets, 2 NAT Gateways.
+#
+# Add variables:
+# VPCName="RunpleVPC"
+# Region="us-east-1" or any from Mappings:AZRegions
+# Environment="Dev" or "Prod"
+#
+# To create/update stack use:
+# aws cloudformation deploy --stack-name ${Environment}${VPCName} --template-file ops/cfn/vpc.yml --parameter-overrides Environment=${Environment} --region ${Region}
+#
+# To delete stack use:
+# aws cloudformation delete-stack --stack-name ${Environment}${VPCName} --region ${Region}
+---
+  AWSTemplateFormatVersion: "2010-09-09"
+
+  Description: "Custom VPC across 2 AZs with 2 public and 4 private subnets"
+
+  Parameters:
+    Environment:
+      Type: String
+      AllowedValues: [ Dev, Prod ]
+      ConstraintDescription: Must be a valid environment name (Dev or Prod)
+
+  Mappings:
+    SubnetConfig:
+      VPC:
+        Dev: "172.16.0.0/16"
+        Prod: "10.0.0.0/16"
+      Public0:
+        Dev: "172.16.0.0/22"
+        Prod: "10.0.0.0/22"
+      Public1:
+        Dev: "172.16.4.0/22"
+        Prod: "10.0.4.0/22"
+      Private0:
+        Dev: "172.16.64.0/18"
+        Prod: "10.0.64.0/18"
+      Private1:
+        Dev: "172.16.128.0/18"
+        Prod: "10.0.128.0/18"
+      Private2:
+        Dev: "172.16.192.0/19"
+        Prod: "10.0.192.0/19"
+      Private3:
+        Dev: "172.16.224.0/19"
+        Prod: "10.0.224.0/19"
+
+    AZRegions:
```

Comment:
I think we need more choices in EU instead of US

Reply:
agree, will do.

Reply:
Done